src/Kinto.elm

```diff
@@ -631,17 +637,30 @@ get resource itemId client =
         |> HttpBuilder.withExpect (Http.expectJson resource.itemDecoder)
 
 
-{-| Create a GET request on one of the plural endpoints. As lists are paginated,
-When the request is succesful, a `Pager` is attached to the reponse message.
+{-| Create a GET request on one of the plural endpoints
 
     getList resource
 
 -}
-getList : Resource a -> Client -> HttpBuilder.RequestBuilder (Pager a)
+getList : Resource a -> Client -> HttpBuilder.RequestBuilder (List a)
 getList resource client =
     endpointUrl client.baseUrl resource.listEndpoint
         |> HttpBuilder.get
         |> HttpBuilder.withHeaders client.headers
+        |> HttpBuilder.withExpect (Http.expectJson resource.listDecoder)
+
+
+{-| Create a GET request on one of the plural endpoints. As lists are paginated,
+When the request is succesful, a `Pager` is attached to the reponse message.
+
+    getPaginatedList resource
+
+-}
+getPaginatedList : Resource a -> Client -> HttpBuilder.RequestBuilder (Pager a)
+getPaginatedList resource client =
```

Comment:
should the `getPaginatedList` helper take the limit? And then we'd remove the ability to provide a `limit` manually (maybe)?